clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/ExceptionClassifier.java

```diff
@@ -29,16 +36,35 @@
 public class ExceptionClassifier {
 
   private final ExceptionClassifierConfigurationProperties properties;
+  private final DynamicConfigService dynamicConfigService;
 
-  public ExceptionClassifier(ExceptionClassifierConfigurationProperties properties) {
+  public ExceptionClassifier(
+      ExceptionClassifierConfigurationProperties properties,
+      DynamicConfigService dynamicConfigService) {
     this.properties = properties;
+    this.dynamicConfigService = dynamicConfigService;
   }
 
   /** Returns whether or not a given Exception is retryable or not. */
   public boolean isRetryable(@Nonnull Exception e) {
     if (e instanceof SpinnakerException) {
       return Optional.ofNullable(((SpinnakerException) e).getRetryable()).orElse(false);
     }
-    return !properties.getNonRetryableClasses().contains(e.getClass().getName());
+
+    String dynamicNonRetryableClasses =
```

Comment:
i saw wrap in a try/catch for extra defensiveness and return `false` if things fail?